resource_manager/unit_tests/test_project.py

```diff
@@ -57,18 +59,22 @@ def test_from_api_repr(self):
         PROJECT_NUMBER = 12345678
         PROJECT_LABELS = {'env': 'prod'}
         PROJECT_LIFECYCLE_STATE = 'ACTIVE'
+        PARENT = {'type': 'organization', 'id': 433637338579}
```

Comment:
FYI: The field type for 'id' is a string.

https://cloud.google.com/resource-manager/reference/rest/v1/projects#ResourceId